agent/consul/state/acl_test.go

```diff
@@ -30,12 +30,7 @@ const (
 )
 
 func setupGlobalManagement(t *testing.T, s *Store) {
```

Comment:
I took a quick look, and I _think_ the answer is no. But wanted to double check with you if it would be useful to add an equivalent `setupBuiltinGlobalReadOnly()` and propagate down the test file similar to `setupGlobalManagement `?